DeveloperGame/DeveloperGame/DeveloperGameClass.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+public class DeveloperGameClass
+{
+    public DeveloperGameClass()
+    {
+        // To do.
+    }
+
+    public void Run()
+    {
+        PrintIntro();
+
+        while (true)
```

Comment:
This is good, but I think the best solution would be:

```
char userMenuChoice;
do
{
    userMenuChoice = GetUserMenuChoice();

    ... Do the rest of your if's except for the quit because it is in the while loop

} while (!UserChoseToQuit);
```

I think it makes it very clear about when we stop our while loop. 
Before the while(true) could be deceiving

Reply:
I fixed this issue.